reference/latest/src/main/resources/data/minecraft/tags/damage_type/bypasses_armor.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Everything should ideally be data generated. Even if its not documented.

Reply:
Tags can be easily done. Do you think damage_types should be data genned as well?

Reply:
Yes, I believe you can just use a codec provider that uses the damage type codec. Correct me if im wrong!

Reply:
Edited comment.

Reply:
Everything is datagenned now. The doc itself will not mention anything about datagen to avoid complexity. Lets add a info box saying this too can be datagenned when the datagen docs are done.